ai/views.py

```
@@ -0,0 +1,210 @@
+from django.shortcuts import render, redirect
```

Ruff (0.8.2)

`django.shortcuts.redirect` imported but unused

Remove unused import: `django.shortcuts.redirect`

(F401)